### PR TITLE
PIMS-1918: Moved LTSA Info to the bottom of Property Details page

### PR DESCRIPTION
**Pull request by Sharala-Perumal**
PIMS-1918:  Move LTSA to bottom of Property Details                                   

## 🎯 Summary

  
[PIMS-1918: ](https://citz-imb.atlassian.net/browse/PIMS-1918)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.

### CODE REVIEW

**Review by LawrenceLau2020 — APPROVED** [review]
LTSA info is now at the bottom, works as expected

File: react-app/src/components/property/PropertyDetail.tsx [hunk 1]
```diff
@@ -351,6 +336,21 @@ const PropertyDetail = (props: IPropertyDetail) => {
...
+            id={'LTSA Information'}
+            values={undefined}
+            title={'LTSA Information'}
+            disableEdit={true}
+            onEdit={undefined}
+          >
+            <TitleOwnership
+              pid={parcel?.parsedBody?.PID ? zeroPadPID(Number(parcel?.parsedBody?.PID)) : null}
+            />{' '}
+            <></>
```
Comment (GrahamS-Quartech) [inline react-app/src/components/property/PropertyDetail.tsx:351]: Nothing to do with your changes but can you remove this `{' '}<></>`. Seems like a mistake from some other PR.

**Review by GrahamS-Quartech — APPROVED** [review]

File: react-app/src/components/property/PropertyDetail.tsx (hunk 1 above)
Comment (Sharala-Perumal) [reply react-app/src/components/property/PropertyDetail.tsx]: Done.

Comment (dbarkowsky) [reply react-app/src/components/property/PropertyDetail.tsx]: I think that auto-formatting, whether through eslint or not, sometimes does this for weird reasons.

**Pull request merged**